### DISCUSSION TITLE
Probleme mit den digits

### DISCUSSION THREAD

**Discussion by nyunity** [jomjol/AI-on-the-edge-device#2677] [General]
### Issue

Hallo, 

mein Wasserzähler hat 4 schwarze Zahlen und drei rote. Die roten sind die Liter und die schwarzen dann die Kubik.

Wie kann ich die Digits jetzt so konfigurieren, dass ich hinten die Liter und vorne die Kubik habe? Bin da echt zu blöd für :-D

Danke,

Erik

**Comment by friedpa** [comment]
Ich weiß zwar nicht, was genau das Problem ist, aber:
- Bei der Einrichtung darauf acht, dass die ROI in der richtigen Reihenfolge sind (durchklicken)
- Mit "Decimal Shift" in der Configuration das Komma verschieben (also -3)
Jetzt solten vorne die m3 sein und nach dem Komma die Liter (also bei drei roten Stellen ist der Wasserzähler dann auf den Liter genau)

**Comment by nyunity** [comment]
hm, so hatte ich es am Anfang, muss ich denn die zwei nullen am anfang auch mit berücksichtigen?

**Comment by friedpa** [comment]
Decimal Shift ist von rechts weg

**Comment by nyunity** [comment]
Schau mal, das habe ich eingestellt, wenn ich von rechts nach links schaue, dann ist dig7 x1, dig6 x10, ...dig1 x1000000 das sind doch dann die Liter... oder?

Habe hier mal meine Einstellungen:
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/8504397/7766f005-27e8-49a9-9f96-e3d9f3d2a973)

![image](https://github.com/jomjol/AI-on-the-edge-device/assets/8504397/91c2c3a7-0a30-49bf-8018-cde1dfe3bab9)

![image](https://github.com/jomjol/AI-on-the-edge-device/assets/8504397/8877d3b5-ce77-4cb5-b0ed-4c35e4f3020b)

Aber er liest auch die letzten Werte falsch... 

Gelesen wird 0649347.6 statt 649818... 

Hast noch ne oder? Erkennbar sind die Zahlen doch recht gut oder?

**Comment by friedpa** [comment]
Du hast den Decimal Shift jetzt bei 0, d.h. derzeit siehst Du Liter angezeigt. Wenn Du in m3 angezeigt haben möchtest, dann Decimal Shift = -3
Das Bild könnte viel besser sein. Bring die Camera näher.
Hier mein Gaszähler als Bespiel, da mein Wasserzähler die letzten Stellen Analog anzeigt:

![grafik](https://github.com/jomjol/AI-on-the-edge-device/assets/42890695/c172ffe9-c5f4-4e63-a9ad-ee8187023843)

**Reply by nyunity** [reply]
Ich habe jetzt nochmal alles neu gemacht, so gar so ausgerichtet, das ich nichts drehen musste auf dem reference Image. Die Marker platziert... auf die Allgin Marker 1 und 2 sind sauber. 

Jetzt guck ich auf "Overview" wo er die Daten liest, ist das Bild wieder verschoben und die Marker sind nicht dort, wo ich sie platziert habe, siehe hier:

![image](https://github.com/jomjol/AI-on-the-edge-device/assets/8504397/b582b68c-5eea-4604-8993-0b29c2023ca8)
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/8504397/e6a043c7-49a0-4944-a668-6516e0ffb653)
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/8504397/fd57277d-82f9-4236-9842-67bf92593e9e)
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/8504397/718d59ff-251c-479b-89ee-2c9de4943948)
